PSKoans/Koans/Advanced Techniques/AboutRedirection.Koans.ps1

```diff
@@ -3,7 +3,140 @@
 param()
 <#
     Redirection
+
+    The PowerShell streams are numbered in order from 1-6.
+
+    Stream #	Description	        Introduced in
+    --------	-----------	        -------------            
+    1	        Success Stream	    PowerShell 2.0
+    2	        Error Stream	    PowerShell 2.0
+    3	        Warning Stream	    PowerShell 3.0
+    4	        Verbose Stream	    PowerShell 3.0
+    5	        Debug Stream	    PowerShell 3.0
```

Comment:
These lines need to be standardized in their indentation levels here, it looks like a few of them ended up a little funky. 😄